Exercise_A/assets/css/style.css

```diff
@@ -1,3 +1,491 @@
+ .navbar-default .nav>li.active>a:focus {
+     color: #f05f40!important;
+     background-color: transparent;
+ }
+ 
+ .navbar-default .nav > li.active>a:hover,
+ .navbar-default .nav>li.active>a:focus:hover {
+     background-color: transparent;
+ }
+ 
+ @media(min-width:768px) {
+     .navbar-default {
+         border-color: rgba(255,255,255,.3);
+         background-color: transparent;
+         border-bottom:1px solid rgba(255,255,255,.3);
+     }
+ 
+     .navbar-default .navbar-header .navbar-brand {
+         color: rgba(255,255,255,.7);
+     }
+ 
+     .navbar-default .navbar-header .navbar-brand:hover,
+     .navbar-default .navbar-header .navbar-brand:focus {
+         color: #fff;
+     }
+ 
+     .navbar-default .nav > li>a,
+     .navbar-default .nav>li>a:focus {
+         color: rgba(255,255,255,.7);
+     }
+ 
+     .navbar-default .nav > li>a:hover,
+     .navbar-default .nav>li>a:focus:hover {
+         color: #fff;
+     }
+    
+     .navbar-default.affix {
+         border-color: rgba(34,34,34,.05);
+         background-color: #fff;
+     }
+ 
+     .navbar-default.affix .navbar-header .navbar-brand {
+         font-size: 14px;
+         color: #f05f40;
+     }
+ 
+     .navbar-default.affix .navbar-header .navbar-brand:hover,
+     .navbar-default.affix .navbar-header .navbar-brand:focus {
+         color: #eb3812;
+     }
+ 
+     .navbar-default.affix .nav > li>a,
+     .navbar-default.affix .nav>li>a:focus {
+         color: #222;
+     }
+ 
+     .navbar-default.affix .nav > li>a:hover,
+     .navbar-default.affix .nav>li>a:focus:hover {
+         color: #f05f40;
+     }
+ }
+ 
+ header {
+     position: relative;
+     width: 100%;
+     min-height: auto;
+     text-align: center;
+     color: #fff;
+     background-image: url(/Exercise_A/assets/images/header.jpg);
+     background-position: center;
+     -webkit-background-size: cover;
+     -moz-background-size: cover;
+     background-size: cover;
+     -o-background-size: cover;
+ }
+ 
+ header .header-content {
+     position: relative;
+     width: 100%;
+     padding: 100px 15px;
+     text-align: center;
+ }
+ 
+ header .header-content .header-content-inner h1 {
+     margin-top: 0;
+     margin-bottom: 0;
+     text-transform: uppercase;
+     font-weight: 700;
+ }
+ 
+ header .header-content .header-content-inner hr {
+     margin: 30px auto;
+ }
+ 
+ header .header-content .header-content-inner p {
+     margin-bottom: 50px;
+     font-size: 16px;
+     font-weight: 300;
+     color: rgba(255,255,255,.7);
+ }
+ 
+ @media(min-width:768px) {
+     header {
+         min-height: 100%;
+     }
+ 
+     header .header-content {
+         position: absolute;
+         top: 50%;
+         padding: 0 50px;
+         -webkit-transform: translateY(-50%);
+         -ms-transform: translateY(-50%);
+         transform: translateY(-50%);
+     }
+ 
+     header .header-content .header-content-inner {
+         margin-right: auto;
+         margin-left: auto;

```

Comment:
consider just using margin: auto

that way you font need to write margin-right / margin-left

Also try to reduce the amount of repeat styles within this stylesheet